app/Limits.cpp

```diff
@@ -17,12 +17,99 @@ Limits::Limits(const std::shared_ptr<const Params>& params)
   : params_(params) {
 }
 
+double Limits::throttleToSpeed(const double throttle) const {
+  double speed_calc;
+  if (throttle < 0) {
+    speed_calc = throttle * params_->velocity_min;
```

Comment:
```suggestion
    speed_calc = std::abs(throttle) * params_->velocity_min;
```

Reply:
Will change to a floor at 0 so worst case negative values cause rover to stop.